vendor/opam/src/core/opamConsole.mli

```diff
@@ -0,0 +1,176 @@
+(**************************************************************************)
```

Comment:
We need to make sure that our code never exercises the code in this module. Perhaps we could replace it with a stub?

Reply:
I've stubbed it out and added a line to the update script to load the stubbed version.